### ISSUE TITLE
Installation of Zonemaster::Engine fails due to unit tests

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#844]
Unit tests fails that earlier passed. We get:

```
# Go to CPANM directory where installation has been tried
$ cd /root/.cpanm/work/1605524244.30417/Zonemaster-Engine-v4.0.2
# Run tests 0-->1 to get more verbose
$ PERL_DL_NONLAZY=0 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(1, 'inc', 'blib/lib', 'blib/arch')" t/Test-dnssec.t
t/Test-dnssec.t .. 
(...)
not ok 244 - afnic.fr gives ALL_ALGO_SIGNED

#   Failed test 'afnic.fr gives ALL_ALGO_SIGNED'
#   at t/Test-dnssec.t line 25.
(...)
not ok 249 - afnic.fr does not give RRSIG_BROKEN

#   Failed test 'afnic.fr does not give RRSIG_BROKEN'
#   at t/Test-dnssec.t line 36.
(...)
not ok 264 - ALGORITHM_UNKNOWN # TODO Need to find/create zones with that error

#   Failed (TODO) test 'ALGORITHM_UNKNOWN'
#   at t/Test-dnssec.t line 309.
not ok 265 - EXTRA_PROCESSING_BROKEN # TODO Need to find/create zones with that error

#   Failed (TODO) test 'EXTRA_PROCESSING_BROKEN'
#   at t/Test-dnssec.t line 311.
not ok 266 - ADDITIONAL_DNSKEY_SKIPPED # TODO Need to find/create zones with that error

#   Failed (TODO) test 'ADDITIONAL_DNSKEY_SKIPPED'
#   at t/Test-dnssec.t line 313.
not ok 267 - NSEC_COVERS_NOT # TODO Need to find/create zones with that error

#   Failed (TODO) test 'NSEC_COVERS_NOT'
#   at t/Test-dnssec.t line 315.
not ok 268 - NSEC_SIG_VERIFY_ERROR # TODO Need to find/create zones with that error

#   Failed (TODO) test 'NSEC_SIG_VERIFY_ERROR'
#   at t/Test-dnssec.t line 316.
not ok 269 - NSEC_NOT_SIGNED # TODO Need to find/create zones with that error

#   Failed (TODO) test 'NSEC_NOT_SIGNED'
#   at t/Test-dnssec.t line 317.
not ok 270 - NSEC3_NOT_SIGNED # TODO Need to find/create zones with that error

#   Failed (TODO) test 'NSEC3_NOT_SIGNED'
#   at t/Test-dnssec.t line 318.
1..270
# Looks like you failed 2 tests of 270.
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/270 subtests 
	(less 2 skipped subtests: 266 okay)

Test Summary Report
-------------------
t/Test-dnssec.t (Wstat: 512 Tests: 270 Failed: 2)
  Failed tests:  244, 249
  Non-zero exit status: 2
Files=1, Tests=270, 89 wallclock secs ( 0.06 usr  0.01 sys + 88.19 cusr  0.26 csys = 88.52 CPU)
Result: FAIL
Failed 1/1 test programs. 2/270 subtests failed.
```

**Comment by vlevigneron** [comment]
Very strange, it should use only recorded data. I'll check that.

le 16 nov., Mats Dufberg a ?crit :
> Unit tests fails that earlier passed. We get:
> 
> ```
> # Go to CPANM directory where installation has been tried
> $ cd /root/.cpanm/work/1605524244.30417/Zonemaster-Engine-v4.0.2
> # Run tests 0-->1 to get more verbose
> $ PERL_DL_NONLAZY=0 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(1, 'inc', 'blib/lib', 'blib/arch')" t/Test-dnssec.t
> t/Test-dnssec.t .. 
> (...)
> not ok 244 - afnic.fr gives ALL_ALGO_SIGNED
> 
> #   Failed test 'afnic.fr gives ALL_ALGO_SIGNED'
> #   at t/Test-dnssec.t line 25.
> (...)
> not ok 264 - ALGORITHM_UNKNOWN # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'ALGORITHM_UNKNOWN'
> #   at t/Test-dnssec.t line 309.
> not ok 265 - EXTRA_PROCESSING_BROKEN # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'EXTRA_PROCESSING_BROKEN'
> #   at t/Test-dnssec.t line 311.
> not ok 266 - ADDITIONAL_DNSKEY_SKIPPED # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'ADDITIONAL_DNSKEY_SKIPPED'
> #   at t/Test-dnssec.t line 313.
> not ok 267 - NSEC_COVERS_NOT # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'NSEC_COVERS_NOT'
> #   at t/Test-dnssec.t line 315.
> not ok 268 - NSEC_SIG_VERIFY_ERROR # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'NSEC_SIG_VERIFY_ERROR'
> #   at t/Test-dnssec.t line 316.
> not ok 269 - NSEC_NOT_SIGNED # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'NSEC_NOT_SIGNED'
> #   at t/Test-dnssec.t line 317.
> not ok 270 - NSEC3_NOT_SIGNED # TODO Need to find/create zones with that error
> 
> #   Failed (TODO) test 'NSEC3_NOT_SIGNED'
> #   at t/Test-dnssec.t line 318.
> 1..270
> # Looks like you failed 2 tests of 270.
> Dubious, test returned 2 (wstat 512, 0x200)
> Failed 2/270 subtests 
> 	(less 2 skipped subtests: 266 okay)
> 
> Test Summary Report
> -------------------
> t/Test-dnssec.t (Wstat: 512 Tests: 270 Failed: 2)
>   Failed tests:  244, 249
>   Non-zero exit status: 2
> Files=1, Tests=270, 89 wallclock secs ( 0.06 usr  0.01 sys + 88.19 cusr  0.26 csys = 88.52 CPU)
> Result: FAIL
> Failed 1/1 test programs. 2/270 subtests failed.
> ```
> 
> 
> -- 
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub:
> https://github.com/zonemaster/zonemaster-engine/issues/844

-- 
	Vincent Levigneron  A.F.N.I.C.  Vincent.Levigneron@afnic.fr

**Comment by matsduf** [comment]
To be resolved by #845
